src/main/java/org/jabref/gui/preferences/AppearanceTabViewModel.java

```diff
@@ -0,0 +1,131 @@
+package org.jabref.gui.preferences;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javafx.beans.property.BooleanProperty;
+import javafx.beans.property.SimpleBooleanProperty;
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+
+import org.jabref.gui.DialogService;
+import org.jabref.gui.util.ThemeLoader;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.preferences.JabRefPreferences;
+
+import de.saxsys.mvvmfx.utils.validation.FunctionBasedValidator;
+import de.saxsys.mvvmfx.utils.validation.ValidationMessage;
+import de.saxsys.mvvmfx.utils.validation.ValidationStatus;
+
+public class AppearanceTabViewModel implements PreferenceTabViewModel {
+
+    private final BooleanProperty fontOverrideProperty = new SimpleBooleanProperty();
+    private final StringProperty fontSizeProperty = new SimpleStringProperty();
+    private final BooleanProperty fontTweaksLinuxProperty = new SimpleBooleanProperty();
+    private final BooleanProperty themeLightProperty = new SimpleBooleanProperty();
+    private final BooleanProperty themeDarkProperty = new SimpleBooleanProperty();
+
+    private final DialogService dialogService;
+    private final JabRefPreferences preferences;
+
+    private FunctionBasedValidator fontSizeValidator;
+
+    private List<String> restartWarnings = new ArrayList<>();
+
+    public AppearanceTabViewModel(DialogService dialogService, JabRefPreferences preferences) {
+        this.dialogService = dialogService;
+        this.preferences = preferences;
+
+        fontSizeValidator = new FunctionBasedValidator<>(
+                fontSizeProperty,
+                input -> {
+                    try {
+                        int fontSize = Integer.parseInt(fontSizeProperty().getValue());
+                        return (fontSize > 8) && (fontSize < 22);
```

Comment:
Why limit the font size?

Reply:
I find that a reasonable interval. Otherwise users set the font size to 1 and then no longer find the setting again to reset it.

Reply:
I am all for a lower limit, but not an upper limit. People with disabilites might need to have larger font size

Reply:
Should we choose a higher limit or just set a lower limit and leave the higher to the user?

Reply:
Just leave the lower limit